lib/from_honeybee/geometry/aperture.rb

```diff
@@ -60,7 +60,11 @@ def to_openstudio(openstudio_model)
       end
       reordered_vertices = OpenStudio.reorderULC(os_vertices)
 
-      os_subsurface = OpenStudio::Model::SubSurface.new(reordered_vertices, openstudio_model)
+      os_subsurface = OpenStudio::Model::SubSurface.new(os_vertices, openstudio_model)
+      os_vertices = os_subsurface.vertices
+      # reorder vertices
+      reordered_vertices = OpenStudio.reorderULC(os_vertices)
+      os_subsurface.setVertices(reordered_vertices)
```

Comment:
Why did you make this change?  It looks like you are now running the reorder command twice and you have turned a single, efficient line of code into 5. Was there some reason that the previous code was not working? If not, can you change it back for all 4 cases that this code appears?